platform/feature-api/communication/src/remote-pod/tests/node/bubblewrap.test.ts

```diff
@@ -0,0 +1,19 @@
+import { podBubblewrapClasses, podBubblewrap } from "../../remote";
+import { NamedNode } from "@polypoly-eu/api";
+
+describe("Classes are accounted for", () => {
+    it("has the right number of classes", () => {
+        expect(Object.keys(podBubblewrapClasses).length).toBe(6);
```

Comment:
when does this number change? 
shouldn't this be mocked to represent real implementation rather than checking a specific number? 🤔

Reply:
Well, the main idea is that this will fail if it changes. Eliminating one from here caused the problem indicated, so we don't want to change it again, or at least to be conscious that if it changes, something is going to fail down the line.

Reply:
Hm, I think this needs quite a bit more context to be useful. Maybe something like: "When removing a bubblewrap class, please make sure that no platform relies on it anymore"

Reply:
I will probably just remove it. Since I'm already testing the six instances individually, this is rather redundant. The initial intention was exactly that, but now if some type is removed, it will break *the other* tests, so maybe just eliminate it?
It will still break if something is added, but then, adding something will not be breaking anything as far as we can tell. That makes it brittle, so better just do away with it.

Reply:
Ah, somehow didn't even notice that - though `testInstances` has 5 entries and you're testing for 6 here - so I guess it's not fully redundant?

Anyway, I think my main point is that, ideally in the message of the failing test, but at least as a source code comment, we should explain why this test even exists, and what it means if it fails, like the example copy I came up with above, or something more elaborate.

Reply:
Quad needs its own test; it's not so easy to create.